pkg/operator/ceph/object/rgw-readiness-probe.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -x
+
+# For the readiness probes, the RGW should be removed from the load balancer (service) pool if
+# clients can't reach it to eliminate unnecessary client I/O failures.
+
+RGW_URL="0.0.0.0:8080"
+SWIFT_HEALTHCHECK_URL="$RGW_URL/swift/healthcheck"
+
+function check() {
+  local URL="$1"
+  # --silent - don't output progress info
+  # --output /dev/stderr - output HTML header to stdout (good for debugging)
+  # --write-out '%{response_code}' - print the HTTP response code to stdout
+  curl --silent --output /dev/stderr --write-out '%{response_code}' "$URL"
+}
+
+RATE_LIMITED_HTTP_CODE=498 # 498 for swift
```

Comment:
```suggestion
RATE_LIMITED_HTTP_CODE=498 # 498 for swift
```
we are overwriting this value at L25

Reply:
Only in the case of error.